### ISSUE TITLE
The link for the STL File location links are big on some pages. Need to make it smaller 

### ISSUE THREAD

**Issue by Venkat4real** [tetrabiodistributed/papra#112]
The STL File location links are big on some page's. For example on this page.
https://tetrabiodistributed.github.io/papra/v1.0/fan-box/
       3D Printed Parts Table
       Gaskets Table 
       Vendor Bill Of Materials Table 
       Recommended Tools Table 
       
smaller hyperlinks looks good and readable. 

![Screenshot from 2023-07-20 20-26-31](https://github.com/tetrabiodistributed/papra/assets/96865121/16800c23-4988-4128-88d9-38947125d0fb)

**Comment by mmroden** [comment]
Maybe instead of hyperlinks, we put in readable text that then links to the appropriate location?  We can't really change the link shape, that's dictated by github (unless we want to have version control that's not git)

**Comment by Venkat4real** [comment]
Yes. just the name of the part will be fine. then it goes to the appropriate location.
